force-app/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -146,6 +146,11 @@
         <field>Event_Registration__c.Registration_Id__c</field>
```

Comment:
@sgoEma do you have any idea why there isn't a flowaccess tag in Admin profile for "Before_Insert_Update_Public_Event" flow that is newly created and included into this PR?

Reply:
Yes, "Before_Insert_Update_Public_Event" flow that is newly created and included into this PR
I'm  checking Admin Profile now

Reply:
Admin Profile is updated.